### ISSUE TITLE
Plugin seems to stop working in PhpStorm 2020.2

### ISSUE THREAD

**Issue by ilya-realforce** [Vektah/CodeGlance#250]

**Comment by AndreKR** [comment]
Not a solution, but I switched to "Code MiniMap".

**Comment by gdude2002** [comment]
Code Minimap no longer exists, it seems.

**Comment by AndreKR** [comment]
Yes, I noticed. :(

And no one (including myself) forked it before it disappeared.

**Comment by jturbide** [comment]
I got it working after installing CodeGlance from the marketplace, and then manually installing this one:
https://github.com/mgziminsky/Minimap-for-Jetbrains

Jetbrain will throw an exception when it will start, then you remove the Minimap plugin and tada! CodeGlance is working :)

**Comment by GsHeri** [comment]
ok but that's no real solution, is it.
and nobody seems to care, so i assume this plugin is dead too?

**Comment by AndreKR** [comment]
It has been for a long time.

**Comment by GsHeri** [comment]
that's a real shame... one of my favorite plugins
and there's still no alternative? :/

**Comment by AndreKR** [comment]
Unless someone has a local copy of the Code MiniMap repository, probably not.

**Comment by etosan** [comment]
So what is the issue? Does Code Glance depends on MiniMap somehow?

**Comment by AndreKR** [comment]
No, it's just that "Code MiniMap" worked properly and CodeGlance does not.

**Comment by etosan** [comment]
Ah it seems "Code MiniMap" is fork of CodeGlancec that still used to work, correct?

**Comment by AndreKR** [comment]
I don't think it was a fork but a completely separate implementation.

**Comment by gennaios** [comment]
I do not really know Java but may attempt someday. Perhaps someone else could try. Maybe looking thru changes in the plugin API, finding what needs to be updated may be enough? 

https://plugins.jetbrains.com/docs/intellij/api-changes-list.html#known-breaking-changes

**Comment by foosel** [comment]
FWIW, this also stopped working in PyCharm 2021.1, but https://plugins.jetbrains.com/plugin/15562-codeglance2 seems to work. Maybe an option for PhpStorm users as well?

**Comment by gdude2002** [comment]
> FWIW, this also stopped working in PyCharm 2021.1, but plugins.jetbrains.com/plugin/15562-codeglance2 seems to work. Maybe an option for PhpStorm users as well?

Holy crap, someone did it. This is working great in IDEA Ultimate for me, so I imagine it'd work for most.

**Comment by bestv5** [comment]
I  rebuild  in my  repo： https://github.com/ranbest/CodeGlance/releases
